river/drift/page_hinkley.py

```diff
@@ -86,12 +88,18 @@ def update(self, value) -> tuple:
 
         self.sample_count += 1
 
+        if self.sum <= self.Mint:
```

Comment:
This is not necessary. We are already accounting for this in line 87. Based on the original paper:

![image](https://user-images.githubusercontent.com/17923265/109223264-3d23ae80-781f-11eb-9536-f840622f47cb.png)

Reply:
oh okay!! Thanks.

I seem to understand why they're equivalent now.
In eq.6, it defines `S'_0 = 0` and enforces the calculation on the positive side. So, the min(S_i) will always be 0 and we don't need to find the `self.Mint` anymore. Because min(S_i) is 0, so eq.5 (s_n - min(S_i) >= h) is equivalent to eq.7.

Do I missing or misunderstanding anything here? Many thanks!

Reply:
That is correct :-)